libbeat/docs/shared-env-vars.asciidoc

```diff
@@ -49,3 +67,36 @@ and what each configuration looks like after replacement:
 |`name: ${NAME:beats}` |no setting            |`name: beats`
 |`name: ${NAME:beats}` |`export NAME=elastic` |`name: elastic`
 |==================================
+
+[float]
+=== Specifying Complex Objects in Environment Variables
```

Comment:
Note: the same syntax for complex objects can be used via `-E` setting as well.

e.g. I can do from command line: `-E output='{elasticsearch.enabled: false, console.pretty: true}'`, to disable the ES output and have all events written to console.